message/ops.go

```diff
@@ -212,9 +212,9 @@ func (op Op) String() string {
 	case HandshakeOp:
 		return "handshake"
 	case GetPeerListOp:
-		return "get_peerlist"
+		return "get_peers"
 	case PeerListOp:
```

Comment:
I think there are still a number of places in the code that reference `PeerList`s. These op types are an example, but I think a number of places in the `network` package need to be renamed as well as the `p2p` proto

Reply:
@MukulKolpe were you still planning on working on this?

Reply:
Yes, I'm on it.